nbdev/quarto.py

```diff
@@ -259,9 +261,9 @@ def nbdev_quarto(
     pys = globtastic(path, file_glob='*.py', **kwargs).filter(_is_qpy)
     for py in pys: _exec_py(py)
     if preview: os.system(f'cd "{path}" && quarto preview --port {port}')
-    else: _sprun(f'cd "{path}" && quarto render')
-    if not preview:
-        nbdev_readme.__wrapped__(path, doc_path)
+    else: 
+        nbdev_readme.__wrapped__(path, doc_path) # README must be generated BEFORE render because of how index.html is generated by Quarto
```

Comment:
Readme should only be generated if not preview

Reply:
Isn't that what the else statement does here?

Reply:
Oh yeah absolutely. Now I'm confused as to why it wasn't done that way before!